mi/dataset/parser/nutnr_b_dcl_conc.py

```diff
@@ -79,8 +84,14 @@ def _create_instrument_particle(self, inst_match):
         instrument match data found from parsing an input file.
         """
 
-        # Obtain the ntp timestamp
-        ntp_timestamp = self._extract_instrument_ntp_timestamp(inst_match)
+        # Obtain the port_timestamp
+        port_timestamp = self._extract_dcl_controller_ntp_timestamp(inst_match)
+
+        # obtain internal_timestamp
+        instrument_timestamp = (inst_match.group(InstrumentDataMatchGroups.INST_GROUP_JULIAN_DATE))
+        internal_timestamp = julian_time_to_ntp(instrument_timestamp) + \
+                             (float(inst_match.group(InstrumentDataMatchGroups.INST_GROUP_TIME_OF_DAY))
```

Comment:
i think this more appropriate since second argument comes in different format example decimal hours, some time decimal seconds with microseconds